demo/universal/native/lib/ServerFunctions.re

```diff
@@ -248,6 +281,20 @@ module Samples = {
   let simpleResponseRouteHandler = _args =>
     simpleResponseHandler() |> Lwt.map(response => React.Json(response));
 
+  // This is the action generated to be used under the hood for the server and client
+  // Melange don't display the comment after compilation
+  // This is a little hack while we don't have the ppx to generate the comment
+  switch%platform () {
+  | Server => ()
+  | Client =>
+    // Comment to esbuild create the manifest
+    [%mel.raw
+     {|
+    // extract-server-function id/samples/simpleResponse simpleResponse Samples
+    ''
+    |}
+    ]
+  };
```

Comment:
While we don't have the ppx, I have to add it with the switch platform, with a little hack adding an empty string cause only `%mel.raw` was working, not the `%%mel.raw`